pkg/apis/crd.projectcalico.org/v1/felixconfig.go

```diff
@@ -213,7 +213,7 @@ type FelixConfigurationSpec struct {
 	// set to false. This reduces the number of metrics reported, reducing Prometheus load. [Default: true]
 	PrometheusProcessMetricsEnabled *bool `json:"prometheusProcessMetricsEnabled,omitempty"`
 	// PrometheusReporterPort specifies the TCP port on which to report denied packet metrics.
-	PrometheusReporterPort *int `json:"prometheusReporterPort"`
+	PrometheusReporterPort *int `json:"prometheusReporterPort,omitempty"`
```

Comment:
Without this the operator was getting errors when trying to create the FelixConfiguration with no PrometheusReporterPort value.

Reply:
This also seems to match what is latest in [tigera/api](https://github.com/tigera/api/blob/master/pkg/apis/projectcalico/v3/felixconfig.go#L311)